src/libopensc/card-epass2003.c

```diff
@@ -319,7 +319,7 @@ gen_init_key(struct sc_card *card, unsigned char *key_enc, unsigned char *key_ma
 	unsigned char iv[16] = { 0 };
 	
 	if (!card->drv_data) 
-		return SC_ERROR_INVALID_ARGUMENTS
+		return SC_ERROR_INVALID_ARGUMENTS;
 	
 	epass2003_exdata *exdata = (epass2003_exdata *)card->drv_data;
```

Comment:
It's better to declare all variables at the beginning of block, before the first non-declaration instruction.